ansible/bootstrap.yml

```diff
@@ -258,3 +258,9 @@
   tasks:
     - include_role:
         name: azimuth_cloud.image_utils.linux_ansible_init
+
+- hosts: k3s
```

Comment:
This PR needs an image with this included - although I know cross-cloud image handling is hard at present.